virttest/remote_commander/remote_runner.py

```diff
@@ -703,6 +711,54 @@ def exit(self):
         self._exit = True
         return "bye"
 
+    def run_test(self, test_path):
+        """Call run() function at external python module.
+
+        Parameters
+        ----------
+        test_path : str
+            Path to python file. Call run() function at this module.
+
+        Returns
+        -------
+            module.run().return
+        """
+        assert os.path.isfile(test_path)
+        module = imp.load_source('ext_test', test_path)
```

Comment:
1. It is the same benefit why avocado-vt requires to use run() in avocado-vt tests.
2. This tests are not supposed be run from command-line.